data/src/main/java/com/example/data/Utils/Converters.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.data.Utils
+
+import androidx.room.TypeConverter
+import com.example.data.database.entities.ListDB
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+
+object Converters {
+    val gson = Gson()
```

Comment:
instead of use gson, I recommend to use the native kotlin converter 

`private val json = Json {
        ignoreUnknownKeys = true
        coerceInputValues = true
    }
    json.asConverterFactory("application/json".toMediaType())`

Reply:
Doing

Reply:
this is still here. Will you make the change? If not, I can resolve the comment

Reply:
I was researching what you suggested, but did not change

Reply:
Done